docs/_static/wharf-ci-schema.json

```diff
@@ -0,0 +1,501 @@
+                                "linux",
+                                "windows"
+                            ]
+                        },
+                        "shell": {
+                            "type": "string",
+                            "examples": [
+                                "/bin/sh"
+                            ]
+                        },
+                        "certificatesMountPath": {
+                            "type": "string",
+                            "examples": [
+                                "/usr/local/share/ca-certificates"
+                            ]
+                        },
+                        "secretName": {
+                            "type": "string",
+                            "examples": [
+                                "spark-system-tests-secret"
+                            ]
+                        }
+                    }
+                },
+                "kubectl": {
+                    "title": "Kubectl build step type",
+                    "description": "Run `kubectl apply -f` on a Kubernetes manifest.",
+                    "type": "object",
+                    "additionalProperties": false,
+                    "required": [
+                        "namespace"
+                    ],
+                    "properties": {
+                        "file": {
+                            "type": "string",
+                            "examples": [
+                                "deploy/api.yaml"
+                            ]
+                        },
+                        "files": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            },
+                            "examples": [
+                                [
+                                    "deploy/api.yaml"
+                                ]
+                            ]
+                        },
+                        "namespace": {
+                            "type": "string",
+                            "examples": [
+                                "stage"
+                            ]
+                        },
+                        "action": {
+                            "type": "string",
+                            "default": "apply",
+                            "examples": [
+                                "apply",
+                                "create",
+                                "patch",
+                                "delete"
+                            ]
+                        },
+                        "force": {
+                            "type": "boolean",
+                            "default": false
+                        },
+                        "cluster": {
+                            "type": "string",
+                            "examples": [
+                                "kubectl-config",
```

Comment:
Maybe here should be different examples.

Reply:
Changed to `stage-cluster-config` and `prod-cluster-config` to mimic #1